modules/post/multi/manage/shell_to_meterpreter.rb

```diff
@@ -335,7 +335,7 @@ def cleanup_handler(listener_job_id, aborted)
   # Show the progress of the upload
   #
   def progress(total, sent)
-    done = (sent.to_f / total.to_f) * 100
+    done = (sent.to_f / total) * 100
```

Comment:
Not a safe change to make automatically, but I verified this is with a number and not a string so it seems okay to me

Reply:
Opting to disable this rule entirely to make things safer to run rubocop over Metasploit without manual verification